modules/ROOT/nav.adoc

```diff
@@ -23,15 +23,17 @@
  ** xref:proxy-deploy-runtime-fabric.adoc[Deploy an API Proxy to Anypoint Runtime Fabric]
 * xref:upgrade-index.adoc[Upgrade Runtime Fabric]
 * xref:manage-index.adoc[Manage Runtime Fabric]
+ ** xref:resource-usage.adoc[View Runtime Fabric Status and Resources]
  ** xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric]
  ** xref:using-opscenter.adoc[Monitor Anypoint Runtime Fabric]
- ** xref:install-patches.adoc[Perform Scheduled Maintenance]
- ** xref:manage-backup-restore.adoc[Backup and Restore Runtime Fabric]
- ** xref:manage-nodes.adoc[Add or Remove a Node from a Runtime Fabric]
  ** xref:install-rtfctl.adoc[Install the Runtime Fabric Command Line Tool]
+ ** xref:install-patches.adoc[Perform Scheduled Maintenance]
+ ** xref:replace-license-key.adoc[Replace a Mule License Key]
  ** xref:manage-proxy.adoc[Manage Proxies used by Runtime Fabric]
  ** xref:manage-secure-properties.adoc[Manage Secure Properties in Runtime Fabric]
  ** xref:config-mutual-auth.adoc[Configure Mutual Authentication with Anypoint Platform]
- ** xref:replace-license-key.adoc[Replace a Mule License Key]
- ** xref:troubleshoot-guide.adoc[Troubleshooting Runtime Fabric]
+ ** xref:manage-nodes.adoc[Add or Remove a Node from a Runtime Fabric]
+ ** xref:manage-backup-restore.adoc[Backup and Restore Runtime Fabric]
```

Comment:
```suggestion
 ** xref:manage-backup-restore.adoc[Back Up and Restore Runtime Fabric]
```
The verb form is two words; in a title, the preposition that is part of a compound verb it init. capped.